documentation/docs/guides/smart-context-management.md

```diff
@@ -367,6 +367,14 @@ Pricing data is regularly fetched from the OpenRouter API and cached locally. Th
 These costs are estimates only, and not connected to your actual provider bill. The cost shown is an approximation based on token counts and public pricing data.
```

Comment:
should we still mention here in the goose desktop section that it will be at the bottom of your goose desktop window?